static/js/components/VideoPlayer.js

```diff
@@ -216,6 +225,34 @@ export default class VideoPlayer extends React.Component<*, void> {
     )
   }
 
+  imageLoaded = (loaded: boolean) => {
```

Comment:
I might suggest naming this function more to reflect what it does - maybe `setVideoSource`  or something like that?

Reply:
sure, I think I'll rename it to `switchVideoSource`